nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```
@@ -178,6 +178,8 @@ nifi.sensitive.props.algorithm=${nifi.sensitive.props.algorithm}
 nifi.sensitive.props.provider=${nifi.sensitive.props.provider}
 nifi.sensitive.props.additional.keys=${nifi.sensitive.props.additional.keys}
 
+nifi.security.autorefresh.enabled=${nifi.security.autorefresh.enabled}
+nifi.security.autorefresh.interval=${nifi.security.autorefresh.interval}
```

Should these properties be named autoreload or just reload to follow the method semantics of the Jetty implementation?  I realize this will involve changes in multiple places.
```suggestion
nifi.security.reload.enabled=${nifi.security.autorefresh.enabled}
nifi.security.reload.interval=${nifi.security.autorefresh.interval}
```